src/sentry/api/endpoints/group_environment_details.py

```diff
@@ -1,85 +0,0 @@
-from __future__ import absolute_import
```

Comment:
can you go into the background for removing this? are we just not using it anymore?